Package.swift

```diff
@@ -22,14 +22,13 @@ let package = Package(
   dependencies: [
     .package(
       name: "abseil",
-      //TODO(yulinliang): Firebase abseil is too old, update the dependency
-      //when abseil repo supports SPM.
       url: "https://github.com/firebase/abseil-cpp-SwiftPM.git",
-      .revision("05d8107f2971a37e6c77245b7c4c6b0a7e97bc99")
+      "0.20220203.0"..<"0.20230101.0"
```

Comment:
It will be safer to make the upper bound 0.20220204.0 so we can allow patch updates, but not automatically roll other 2022 releases into existing deployments

Reply:
updated